### PR TITLE
Create draft of logic_op_dynamic_state sample

### PR DESCRIPTION
**Pull request by Karina-Murawko-Wisniewska-Mobica**

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -271,6 +271,10 @@ Demonstrate how to do conditional rendering, dynamically discarding rendering co
 **Extension**: [```VK_EXT_vertex_input_dynamic_state```](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_vertex_input_dynamic_state.html) <br/>
 Demonstrate how to use vertex input bindings and attribute descriptions dynamically, which can reduce the number of pipeline objects that are need to be created.
 
+### Logic operations dynamic state](./extensions/logic_op_dynamic_state)
+**Extension**: [```VK_EXT_extended_dynamic_state2```](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_extended_dynamic_state2.html) <br/>
+Demonstrate how to use logical operations dynamically, which can reduce the number of pipeline objects that are needed to be created or allow to change the pipeline state dynamically (change type of the logical operation) .
```
Comment (Szymon-Rajczakowski) [inline samples/README.md]: Remove space before dot.

File: samples/extensions/logic_op_dynamic_state/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+get_filename_component(FOLDER_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
+get_filename_component(PARENT_DIR ${CMAKE_CURRENT_LIST_DIR} PATH)
+get_filename_component(CATEGORY_NAME ${PARENT_DIR} NAME)
+
+add_sample(
+    ID ${FOLDER_NAME}
+    CATEGORY ${CATEGORY_NAME}
+    AUTHOR "Mobica"
+    NAME "Logic operations dynamic state"
+    DESCRIPTION "Demonstrate how to use logic operations dynamically, which can reduce the number of pipeline objects that are need to be created."
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/logic_op_dynamic_state/CMakeLists.txt]: are needed

File: samples/extensions/logic_op_dynamic_state/README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+In the sample two pipelines are created: 
+
+- a pipeline for a background model (the background pipeline);
+- a pipeline for a cube model (the baseline pipeline). 
+
+The cube model is used to present blending effect done using the dynamic logical operations.
+
+`logicOpEnable` in `VkPipelineColorBlendStateCreateInfo` structure [controls if logical operations should be applied](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPipelineColorBlendStateCreateInfo.html).
+
+`logicOpEnable` is set to `VK_TRUE` during creation of the baseline pipeline in `create_pipeline` method.
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/logic_op_dynamic_state/README.md]: I think it should be:
(...) in the `create_pipeline` method.

File: samples/extensions/logic_op_dynamic_state/README.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+```C++
+VkPipelineColorBlendStateCreateInfo color_blend_state =
+    vkb::initializers::pipeline_color_blend_state_create_info(1, &blend_attachment_state);
+
+/* Enable logic operations */
+color_blend_state.logicOpEnable = VK_TRUE;
+```
+
+In the same method `VK_DYNAMIC_STATE_LOGIC_OP_EXT` is added to vector `VkDynamicState`.
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/logic_op_dynamic_state/README.md]: If think it should be:
(...) is added to the `VkDynamicState` vector.

**Review by Szymon-Rajczakowski — CHANGES_REQUESTED** [review]

File: samples/extensions/logic_op_dynamic_state/README.md (hunk 4 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/logic_op_dynamic_state/README.md]: fixed

File: samples/extensions/logic_op_dynamic_state/README.md (hunk 3 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/logic_op_dynamic_state/README.md]: fixed

File: samples/extensions/logic_op_dynamic_state/CMakeLists.txt (hunk 2 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/logic_op_dynamic_state/CMakeLists.txt]: fixed

File: samples/README.md (hunk 1 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/README.md]: fixed

**Review by Szymon-Rajczakowski — APPROVED** [review]

File: samples/README.md [hunk 5]
```diff
@@ -271,6 +271,10 @@ Demonstrate how to do conditional rendering, dynamically discarding rendering co
 **Extension**: [```VK_EXT_vertex_input_dynamic_state```](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_EXT_vertex_input_dynamic_state.html) <br/>
 Demonstrate how to use vertex input bindings and attribute descriptions dynamically, which can reduce the number of pipeline objects that are need to be created.
 
+### Logic operations dynamic state](./extensions/logic_op_dynamic_state)
```
Comment (Zbigniew-Jedlinski-Mobica) [inline samples/README.md]: There missing "[" character before "Logic" word

Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/README.md]: fixed

**Pull request merged**